dev/devicelab/manifest.yaml

```diff
@@ -510,6 +510,20 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios"]
 
+  smoke_catalina_start_up:
+    description: >
+      A smoke test that runs on macOS Catalina, which is a clone of the Gallery startup latency test.
+    stage: devicelab_ios
+    flaky: true
+    required_agent_capabilities: ["mac-catalina/ios"]
+
+  smoke_catalina_hot_mode_dev_cycle__benchmark:
+    description: >
+      A some test that runs on macOS Catalina, which is a clone of the Dart VM hot patching performance benchmarking.
```

Comment:
Oh, hmm.. will this automatically spawn a new benchmark series?  That's ok, I guess.